homeassistant/components/aruba/device_tracker.py

```diff
@@ -24,6 +31,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_HOST): cv.string,
+        vol.Optional(CONF_MODE, default=DEFAULT_MODE): cv.string,
```

Comment:
We no longer allow changes to YAML schemas of integration that communicate with a device or service.

See ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
And ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md>

This integration needs to be converted to use the configuration flow first. See our developer documentation on how to get started on that.